### PR TITLE
Add English locale to bot

### PR DESCRIPTION
**Pull request by jDan735**

### CODE REVIEW

File: bot/i18n/errors.en.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+en:
+  enter_wiki_query: "*Please, enter name of article*\nFor example: `{}` Name of article"
+  duckerror: "*Please, enter query*\nFor example: `{}` Name of article"
+  few_args: "*Insufficient number of parameters:*\nRequired `%{num}`"
+
+  message_len: "*Telegram support messages with a maximum length of 4096 characters*"
```
Comment (0x0bloodyknight) [inline bot/i18n/errors.en.yml:6]: supports

File: bot/i18n/menu.en.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
+en:
+    main: |
+        <b>✋ Hello!</b> I multifunctional telegram bot for banning and stuff :) Select the commands at the bottom by first clicking on the buttons. If the bot does not work, then write <a href="t.me/jdan734">jDan734</a>
```
Comment (0x0bloodyknight) [inline bot/i18n/menu.en.yml:3]: I'm a multifunctional. можно и без first

File: bot/i18n/menu.en.yml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        /title - all in title case
+        /upper - all in upper case
+        /lower - all in lower case
+
+    images: |
+        <b>Photo editing</b>
+
+        /resize <code>[x] [y]</code> - change photo's size
+        /text <code>[size] [color] [XxY] [text]</code> - draw text in photo
+        /rectangle - draw rectangle in photo
```
Comment (0x0bloodyknight) [inline bot/i18n/menu.en.yml:35]: draw on photo

File: bot/i18n/menu.en.yml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+    commands: |
+        <b>System commands</b>
+
+        /delete - delete replied message
+        /to_json - str(dict) => json
+        /status - bot's status
```
Comment (0x0bloodyknight) [inline bot/i18n/menu.en.yml:42]: bot status

File: bot/i18n/notes.en.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+en:
+  enter_note_name: Enter name of note to delete
```
Comment (0x0bloodyknight) [inline bot/i18n/notes.en.yml:2]: of the note

File: bot/i18n/menu.en.yml (hunk 4 above)
Comment (jDan735) [reply bot/i18n/menu.en.yml:42]: фiксуемо ~прибуток~ помилки

**Pull request merged**